core/src/main/java/org/web3j/protocol/rx/Web3jRx.java

```diff
@@ -25,39 +23,39 @@
      * @param ethFilter filter criteria
      * @return Observable that emits all Log events matching the filter
```

Comment:
Javadocs need to be updated to say flowable

Reply:
Sorry - I was merely merging the approved PR. I'll do this now.